### PR TITLE
Add Prettier/build check as a CI step

### PR DESCRIPTION
**Pull request by stevenpetryk**
Noticed that Prettier wasn't ran in CI. This will prevent unformatted code. I tried to model the action on `primer`'s GA config.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Prettier
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
```
Comment (stevenpetryk) [inline .github/workflows/ci.yml:20]: `--frozen-lockfile` ensures that anyone who opens a PR has ran `yarn install` locally. Prevents `yarn.lock` from getting out of sync in `main`.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+      - name: Run prettier
+        run: yarn lint
+
+  prettier:
```
Comment (colebemis) [inline .github/workflows/ci.yml:38]: What's the difference between the `eslint` and `prettier` steps?

**Comment by stevenpetryk** [comment]
@colebemis thanks for the feedback! Put all the jobs in one job. And the `eslint` job was a mistake.

**Review by colebemis — APPROVED** [review]
Thanks for setting this up, @stevenpetryk! 🙌

**Pull request merged**